javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -276,6 +276,14 @@ object Character {
   def isUpperCase(c: scala.Char): scala.Boolean =
     reIsUpper.test(c.toString)
 
+  def isValidCodePoint(codePoint: Int): scala.Boolean = {
+    MIN_CODE_POINT <= codePoint && MAX_CODE_POINT >= codePoint
+  }
```

Comment:
Oh and write constants on the right hand side of comparison operators, so like this:
```scala
codePoint >= MIN_CODE_POINT && codePoint <= MAX_CODE_POINT
```